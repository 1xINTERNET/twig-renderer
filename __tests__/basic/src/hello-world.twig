<h1>Hello {{ text }}!</h1>
{% include 'bye-world.twig' %}
