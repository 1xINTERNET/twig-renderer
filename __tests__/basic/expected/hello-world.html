<h1>Hello World!</h1>
<h6>Bye World!</h6>
<footer>I'm a footer</footer>
